src/StaticWebAssetsSdk/Tasks/ComputeReferenceStaticWebAssetItems.cs

```diff
@@ -63,6 +65,7 @@ public override bool Execute()
                 if (ShouldIncludeAssetAsReference(selected, out var reason))
                 {
                     selected.SourceType = UpdateSourceType ? StaticWebAsset.SourceTypes.Project : selected.SourceType;
+                    selected.OriginalItemSpec = MakeReferencedAssetOriginalItemSpecAbsolute ? Path.GetFullPath(selected.OriginalItemSpec) : selected.OriginalItemSpec;
```

Comment:
Not really important, but is this written as a ternary just to avoid taking up more lines with an `if` statement? It just took me a second to parse that only the "then" expression incurs a mutation. I see that the line above is already written the same way, so I understand aiming to follow the existing pattern. Feel free to keep as-is, just giving my two cents.